components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.call.ui/src/main/resources/web/call-mediator/update-mediator.jsp

```diff
@@ -79,7 +79,25 @@
         callMediator.setEndpoint(ep);
     }
 
-    callMediator.setBlocking(Boolean.parseBoolean(request.getParameter("mediator.call.blocking")));
+    //Sets blocking information
+    String param=request.getParameter("mediator.call.blocking");
```

Comment:
Formatting issue: there should be a space before and after =

Reply:
Fixed